app/tools/android/make_apk.py

```diff
@@ -549,7 +560,7 @@ def main(argv):
   parser.add_option('--mode', choices=('embedded', 'shared'),
                     default='embedded', help=info)
   info = ('The target architecture of the embedded runtime. Supported values '
```

Comment:
You could use `AllArchitectures()` here too so you don't need to hardcode the list.

```python
info = ('The target architecture of the embedded runtime. Supported values: %s. If not specified, APKs for all available architectures will be generated.' % ', '.join(AllArchitectures()))
```